assets/component-card.css

```diff
@@ -155,13 +155,26 @@
   z-index: 1;
 }
 
-.card-wrapper:focus-within .card {
-  box-shadow: 0 0 0 .3rem rgb(var(--color-background)),0 0 .5rem .4rem rgba(var(--color-foreground),.3);
+.card__heading a:after {
   outline-offset: 0.3rem;
+}
+
+.card__heading a:focus:after {
+  box-shadow: 0 0 0 .3rem rgb(var(--color-background)),0 0 .5rem .4rem rgba(var(--color-foreground),.3);
+  outline: .2rem solid rgba(var(--color-foreground),.5);
+}
+
+.card__heading a:focus-visible:after {
+  box-shadow: 0 0 0 .3rem rgb(var(--color-background)),0 0 .5rem .4rem rgba(var(--color-foreground),.3);
```

Comment:
When using the card style card [the outline overlaps](https://screenshot.click/06-34-24bgi-cw8z7.png). Might need to add some `.card--card a:after` styles to account for the border. Probably something like this.
```
width: calc(100% + var(card-border-width * 2);
left: calc(var(card-border-width) * -1);
etc..
```